eng/Versions.props

```diff
@@ -207,6 +207,16 @@
     <!-- ILLink -->
     <MicrosoftNETILLinkTasksVersion>7.0.100-1.22518.2</MicrosoftNETILLinkTasksVersion>
     <MicrosoftNETILLinkAnalyzerPackageVersion>$(MicrosoftNETILLinkTasksVersion)</MicrosoftNETILLinkAnalyzerPackageVersion>
+    <!-- Mono Cecil -->
+    <MicrosoftDotNetCecilVersion>0.11.4-alpha.22524.1</MicrosoftDotNetCecilVersion>
+    <MicrosoftDotNetCecilPdbVersion>0.11.4-alpha.22524.1</MicrosoftDotNetCecilPdbVersion>
+    <!-- ILLink dependencies -->
+    <MicrosoftBuildFrameworkVersion>17.0.0-preview-21267-01</MicrosoftBuildFrameworkVersion>
+    <MicrosoftBuildUtilitiesCoreVersion>17.0.0-preview-21267-01</MicrosoftBuildUtilitiesCoreVersion>
+    <MicrosoftDotNetApiCompatVersion>8.0.0-beta.22473.1</MicrosoftDotNetApiCompatVersion>
+    <MicrosoftCodeAnalysisCSharpAnalyzerTestingXunitVersion>1.0.1-beta1.*</MicrosoftCodeAnalysisCSharpAnalyzerTestingXunitVersion>
+    <MicrosoftCodeAnalysisBannedApiAnalyzersVersion>3.3.2</MicrosoftCodeAnalysisBannedApiAnalyzersVersion>
+    <MicrosoftILVerificationVersion>7.0.0-preview.7.22375.6</MicrosoftILVerificationVersion>
```

Comment:
Should not this come from global tools version? Or as it's used by tests only take project dependency?

Reply:
End up using a new runtime-tools-tests.yml for running the tools.linkertests subset

Reply:
You mean adding https://github.com/dotnet/runtime/tree/6756782ade49d053cdcf3cc076bcaa35627b1fb6/src/coreclr/tools/ILVerification 
 whenever we consume MicrosoftILVerificationVersion?

Reply:
We can use live build of ILVerification by replacing `<PackageReference` with `<ProjectReference` pointing to `$(CoreClrProjectRoot)tools\ILVerification\ILVerification.csproj` (unlike others, it's a standalone project).

Reply:
Yep, project reference could do it

Reply:
Agree with @am11. We should use live dependencies whenever possible.